skaffold.yaml

```diff
@@ -8,58 +8,39 @@ build:
       template: "{{.VERSION}}"
   artifacts:
   - image: valory/open-autonomy-open-aea
-    context: deployments/Dockerfiles/open_aea/
+    context: autonomy/data/agent/
     docker:
       dockerfile: Dockerfile
       buildArgs:
-        AEA_VERSION: "1.14.0"
+        AEA_VERSION: "1.14.0.post1"
         AEA_AGENT: "{{.AEA_AGENT}}"
 
 
 profiles:
-  - name: docs
+  - name: release
     build:
-      tagPolicy:
-        envTemplate:
-          template: "{{.VERSION}}"
-      artifacts:
-      - image: valory/open-autonomy-docs
-        docker:
-          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
-  - name: prod
-  - name: dependencies
-    build:
-      tagPolicy:
-        envTemplate:
-          template: "{{.VERSION}}"
-      artifacts:
-      - image: valory/open-autonomy-tendermint
-        context: deployments/Dockerfiles/localnode/
-        docker:
-          dockerfile: Dockerfile
-      - image: valory/open-autonomy-hardhat
-        context: deployments/Dockerfiles/hardhat
-  - name: cluster
-    patches:
-      - op: add
-        path: /build/artifacts/0/kaniko
-        value:
-          buildArgs:
-            AEA_VERSION: "1.14.0"
-            AEA_AGENT: "{{.AEA_AGENT}}"
-      - op: add
-        path: /build/cluster
-        value:
-          pullSecretName: regcred
-          dockerConfig:
-            path: ~/.docker/config.json
-      - op: remove
-        path: /build/artifacts/0/docker
-  - name: dev
-    patches:
-      - op: replace
-        path: /build/artifacts/0/docker/dockerfile
-        value: Dockerfile.develop
-      - op: add
-        path: /build/artifacts/0/docker/cliFlags
-        value: ["--no-cache", "--pull", "--force-rm"]
+        tagPolicy:
+          envTemplate:
+            template: "{{.VERSION}}"
+        artifacts:
+        - image: valory/open-autonomy-open-aea
+          context: autonomy/data/agent/
+          docker:
+            noCache: true
+            dockerfile: Dockerfile
+            buildArgs:
+              AEA_VERSION: "1.14.0.post1"
+              AEA_AGENT: "{{.AEA_AGENT}}"
```

Comment:
Here I'm using the same now..should be fine?

Reply:
I didn't understand why we need hello workd elsewhere and my_first_aea here,,,

Reply:
We can just one one, its just important to have one of the hashes here. The last thing we want to do is push images tagged as latest that *dont* build. or the worst docs which dont have working hashes.